docs_src/microservices/application/AdvancedTopics.md

```diff
@@ -6,261 +6,234 @@ The following items discuss topics that are a bit beyond the basic use cases of
+#### EDGEX_STARTUP_INTERVAL
 
-!!! note
-    This environment variable override has been deprecated in the Geneva Release. Instead, use configuration overrides of **SERVICE_PROTOCOL** and/or **SERVICE_HOST** and/or **SERVICE_PORT**
+This environment variable overrides the retry interval or sleep time before a failure is retried during the start-up, aka bootstrap, phase of execution.
 
-```
-EDGEX_SERVICE=http://192.168.1.2:4903
+#### EDGEX_CONF_DIR
 
-This sets the Service information fields as follows:
-    Protocol: http
-    Host: 192.168.1.2
-    Port: 4903
-```
+This environment variable overrides the configuration directory where the configuration file resides. Default is `./res` and also overrides any value set with the `-c/--confdir` command-line option.
 
-#### EDGEX_PROFILE
+#### EDGEX_CONFIG_FILE
 
-This environment variable overrides the command line `profile` argument. It will set the `profile` or replace the value passed via the `-p` or `--profile`, if one exists. This is useful when running the service via docker-compose.
+This environment variable overrides the configuration file name. Default is `configutation.toml` and also overrides any value set with the -f/--file command-line option.
 
-!!! note
-    The lower case version has been deprecated in the Geneva release. Instead use upper case version **EDGEX_PROFILE**
+### Custom Configuration
 
-Using docker-compose:
+Applications can specify custom configuration in the TOML file in two ways. 
 
-```
-  app-service-configurable-rules:
-    image: edgexfoundry/docker-app-service-configurable:1.1.0
-    environment: 
-      - EDGEX_PROFILE : "rules-engine"
-    ports:
-      - "48095:48095"
-    container_name: edgex-app-service-configurable
-    hostname: edgex-app-service-configurable
-    networks:
-      edgex-network:
-        aliases:
-          - edgex-app-service-configurable
-    depends_on:
-      - data
-      - command
-```
+#### Application Settings
 
-This sets the `profile` so that the application service uses the `rules-engine` configuration profile which resides at `/res/rules-engine/configuration.toml`
+The first simple way is to add items to the `ApplicationSetting` section. This is a map of string key/value pairs, i.e. `map[string]string`. Use for simple string values or comma separated list of string values. The `ApplicationService` API provides the follow access APIs for this configuration section:
 
-!!! note
-    EdgeX services no longer use docker profiles. They use Environment Overrides in *the docker compose file to make the necessary changes to the configuration for running in Docker. See the **Environment Variable Overrides For Docker** section in the [App Service Configurable](./AppServiceConfigurable.md#environment-variable-overrides-for-docker) section for more details and an example.
+- `ApplicationSettings() map[string]string`
+    - Returns the whole map
```

Comment:
we know it is map as it has already been stated above, no need to mention map here.

```suggestion
    - Returns the whole list of application settings
```